scout/server/blueprints/variants/controllers.py

```diff
@@ -233,58 +312,33 @@ def get_manual_assessments(variant_obj):
     for assessment_type in assessment_keywords:
         assessment = {}
         if variant_obj.get(assessment_type) is not None:
-            if assessment_type == "manual_rank":
```

Comment:
I've moved most of this function to another function to lower its complexity (codefactor was complaining)